src/ansys/motorcad/core/rpc_client_core.py

```diff
@@ -428,12 +428,18 @@ def _find_free_motor_cad(self):
                 )
 
     def ensure_version_at_least(self, required_version):
+        """Ensure that the Motor-CAD version is later or equal to required version."""
+        if not self.check_version_at_least(required_version):
+            raise MotorCADError(
+                "This function requires Motor-CAD version: " + required_version + " or later"
```

Comment:
Message strings should always have concluding punctuation, which is a period or full stop. The parameter "required_version" isn't defined in a "Parameters" list section. Is the return value a Boolean?

Reply:
The code in this class should only ever be modified/used by Motor-CAD dev team so happy to leave as is.